Library/Homebrew/cmd/update.rb

```diff
@@ -181,8 +181,20 @@ def report
         next unless paths.any? { |p| File.dirname(p) == formula_directory }
 
         case status
-        when "A", "M", "D"
+        when "A", "D"
           map[status.to_sym] << repository.join(src)
+        when "M"
+          file = repository.join(src)
+          begin
+            require "formula_versions"
```

Comment:
I kinda want to keep this as it is to minimum the chance that we accidentally break `brew update`, which has far more impact than breaking other things.